.travis.yml

```diff
@@ -26,117 +31,112 @@ env:
         - NODE_ENV=test
         - NPM_CONFIG_PROGRESS=false
         - COZY_DESKTOP_HEARTBEAT=1000
-services:
-    - docker
-cache:
-    yarn: true
-    directories:
-        - $HOME/.cache/electron
-        - $HOME/.cache/electron-builder
-        - $HOME/bin
-addons:
-    apt:
-        sources:
-            - ubuntu-toolchain-r-test
-        packages:
-            - gcc-4.8
-            - g++-4.8
+          #- CXX=$([ "$TRAVIS_OS_NAME" = "linux" ] && echo "g++-4.8" || echo $CXX)
 
-# install cozy stack for integration test
-before_install:
+before_install: | # install cozy stack for integration test
```

Comment:
We can move all the build_install to a script to be consistent with other steps

Reply:
Actually I tried and it wouldn't work because the `travis_retry` function doesn't seem to be available inside external bash scripts :(

Reply:
OK, it's a good reason.

For the record, https://twitter.com/plexus/status/499194992632811520 says that you can copy the `travis_retry` function.

Reply:
I would definitely prefer to see the setup in another file. What would be best in your opinion? Copying the function into a `travis_helpers.sh` file or copying the entire `header.sh` file from the travis repo?

Reply:
Just a copy of the function, with a link in a comment from where it's taken